include/defaults.h

```diff
@@ -86,6 +86,7 @@
 #define MQTT_HASS_RETAIN true
 #define MQTT_HASS_TOPIC "homeassistant/"
 #define MQTT_HASS_INDIVIDUALPANELS false
+#define MQTT_HASS_LEGACYNAMES true
```

Comment:
I think this should be the new default at least for new installations. Is it be possible to activate this for installed instances that are updated but deactivate for fresh installations?